build.gradle

```diff
@@ -1,21 +1,22 @@
 buildscript {
     ext {
-        support_version = "24.2.1"
-        dagger_version = "2.6.1"
+        support_version = "25.1.0"
+        dagger_version = "2.8"
```

Comment:
would be OK with this - just as a note - currently Kotlin and dagger 2.8 are not playing well together - so I usually stick to 2.7 currently